pkg/sidecar/config/config.go

```diff
@@ -296,6 +302,22 @@ func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, s
 	return nil
 }
 
+func setupIOTuneCache() error {
+	_, err := os.Stat(scyllaIOPropertiesPath)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return err
+	}
+
+	if errors.Is(err, os.ErrNotExist) {
```

Comment:
can you reshuffle the logic so you don't check the same error twice?

Reply:
done